packages/expo-updates/e2e/__tests__/fixtures/Updates-basic.e2e.js

```diff
@@ -153,7 +165,12 @@ describe('Basic updates e2e', () => {
     jest.setTimeout(300000 * TIMEOUT_BIAS);
     const bundleFilename = 'bundle2.js';
     const newNotifyString = 'test-update-2';
-    const hash = await copyBundleToStaticFolder(updateDistPath, bundleFilename, newNotifyString);
+    const hash = await copyBundleToStaticFolder(
+      updateDistPath,
+      bundleFilename,
+      newNotifyString,
+      platform
+    );
     const assets = await Promise.all(
       [
         'lubo-minar-j2RgHfqKhCM-unsplash.jpg',
```

Comment:
sorry because github does not allow commenting on new binary file, so i comment here instead. the `packages/expo-updates/e2e/__tests__/fixtures/project_files/e2e/tests/assets/lubo-minar-j2RgHfqKhCM-unsplash.jpg` is new file. is the old `packages/expo-updates/e2e/__tests__/assets/lubo-minar-j2RgHfqKhCM-unsplash.jpg` still be used? could we move the file rather than copy a new one?

Reply:
I will check that this is possible